torchvision/io/_video_opt.py

```diff
@@ -45,7 +44,7 @@ class VideoMetaData:
         "audio_duration": float,
         "audio_sample_rate": float,
     }
-    __slots__ = [
+    __slots__: List[str] = [
```

Comment:
Does `mypy` complain here if we drop the annotation? It should be able to deduce it automatically.

Reply:
Oh you're correct, I think I remember I had to in the beginning, but that problem seems to have vanished (many commits on main since then I guess)